entries/wrapAll.xml

```diff
@@ -14,8 +14,7 @@
   <signature>
     <added>1.4</added>
     <argument name="function" type="Function">
-      <desc>A callback function returning the HTML content or jQuery object to wrap around the matched elements. Receives the index position of the element in the set as an argument. Within the function, <code>this</code> refers to the current element in the set.</desc>
-      <argument name="index" type="Integer" />
+      <desc>A callback function returning the HTML content or jQuery object to wrap around all the matched elements. Within the function, <code>this</code> refers to the first element in the set. <strong>Prior to jQuery 3.0</strong>, the callback got called for every element in the set and received the index position of the element in the set as an argument.</desc>
```

Comment:
How about changing "the callback got called" to "the callback was incorrectly called" because it was the wrong behavior.